kclvm/runner/build.rs

```diff
@@ -0,0 +1,66 @@
+fn main() {
```

Comment:
Pls leave a source info about files you copied.

Reply:
> Pls leave a source info about files you copied.

Have added documents to explain it.